src/GoTrueClient.ts

```diff
@@ -141,55 +141,59 @@ export default class GoTrueClient {
 
   /**
    * Creates a new user.
-   * @type UserCredentials
    * @param options.redirectTo The redirect URL attached to the signup confirmation link. Does not redirect the user if it's a mobile signup.
```

Comment:
change to emailRedirectTo

Reply:
whoops, small bug, made it consistent with the sign-in methods and also made the types tighter in the process